### PR TITLE
update and cleanup of book docs

### PR DESCRIPTION
**Pull request by Prajyot-Parab**
**What this PR does / why we need it**:
- update and cleanup of book docs

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

/area provider/ibmcloud

1. Please confirm that if this PR changes any image versions, then that's the sole change this PR makes.

**Release note**:

```release-note
update and cleanup of book docs
```

### CODE REVIEW

**Review by Amulyam24 — APPROVED** [review]
/lgtm

File: docs/book/src/topics/powervs/prerequisites.md [hunk 1]
```diff
@@ -9,8 +9,6 @@
 4. An appropriately configured [Go development environment](https://golang.org/doc/install)
 5. Install `clusterctl` tool (see [here](https://cluster-api.sigs.k8s.io/user/quick-start.html#install-clusterctl))
 6. Install `pvsadm` tool (see [here](https://github.com/ppc64le-cloud/pvsadm#installation))
-6. Install `ibmcloud` tool (see [here](https://github.com/IBM-Cloud/ibm-cloud-cli-release#downloads))
```
Comment (Karthik-K-N) [inline docs/book/src/topics/powervs/prerequisites.md:12]: I think we should mention how to get capibmadm cli tool as well.

Comment (Prajyot-Parab) [reply docs/book/src/topics/powervs/prerequisites.md:12]: Fixed.

**Comment by mkumatag** [comment]
> /lgtm

relying on the @Karthik-K-N 's review..

/approve

**Pull request merged**